Packs/SSLLabs/Integrations/SSLLabs/SSLLabs.yml

```diff
@@ -0,0 +1,149 @@
+category: Vulnerability Management
+commonfields:
+  id: SSL Labs
+  version: -1
+configuration:
+- additionalinfo: 'The registered email address that will be used to access SSL Labs. '
+  display: Registered Email Address
+  name: email
+  required: false
+  type: 0
+description: 'Analyze a host or a URL. '
+display: SSL Labs (Community Contribution)
```

Comment:
Lets make it just 'SSL Labs' as the support level keyword ('community') should not be contained in integration's name.